snuba/consumers/consumer_builder.py

```diff
@@ -97,14 +97,6 @@ def __init__(
             topic, bootstrap_servers=kafka_params.bootstrap_servers
         )
         logger.info(f"librdkafka log level: {self.broker_config.get('log_level', 6)}")
-        self.producer_broker_config = build_kafka_producer_configuration(
-            topic,
-            bootstrap_servers=kafka_params.bootstrap_servers,
-            override_params={
-                "partitioner": "consistent",
-                "message.max.bytes": 50000000,  # 50MB, default is 1MB
-            },
```

Comment:
How come you dropped these parameters? Do we not need them?

Reply:
Right, good point. I wasn't too sure whether we wanted to keep them. Do you have any insight?

Reply:
Added them back to replacements producer (multistorage consumer passes them in for replacements).